ci/preamble.sh

```diff
@@ -36,19 +36,15 @@ bash ci/check-if-docs-pr.sh && exit 0
 echo "--- detect missing license headers"
 make check-license-headers
 
+echo "--- lint codebase: quick feedback"
+make lint-codebase
```

Comment:
The original reason to put this after `make cli-tsc` was that linting errors were often boring and not so insightful, and tsc errors were more exciting, authoritative, and I wanted to see them _even if_ linting failed. Maybe we can still achieve this later. I am ok with moving this :) Especially now that linting feedback is getting better and better.

Reply:
and the implicit `yarn install` within `make cli-tsc` now probably needs to be pulled out / up so that `make lint-codebase` has all its dependencies.

Reply:
> Maybe we can still achieve this later. I am ok with moving this :) Especially now that linting feedback is getting better and better.

Figured we wanted to fail as fast as possible.

> and the implicit yarn install within make cli-tsc now probably needs to be pulled out / up so that make lint-codebase has all its dependencies.

Looks like it.... forgot to test in a clean checkout.